fatiando/seismic/tests/test_seismic_srtomo.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
```

Comment:
On the first line, use the following imports for future Python 3 compatibility:

    from __future__ import division, print_function
    from future.builtins import range

This way you don't need to put `.` in all your numbers "`1./2.`"